manifests/install.pp

```diff
@@ -40,9 +40,9 @@
 
   if !$::vault::disable_mlock {
     exec { "setcap cap_ipc_lock=+ep ${vault_bin}":
-      path        => ['/sbin', '/usr/sbin'],
-      subscribe   => File[$vault_bin],
-      refreshonly => true,
+      path      => ['/sbin', '/usr/sbin', '/usr/bin', ],
+      subscribe => File[$vault_bin],
+      unless    => 'getcap /usr/local/bin/vault | grep cap_ipc_lock+ep',
```

Comment:
This path would need to reference the `$vault_bin` variable rather than a hardcoded path.